apps/system/test/marionette/lib/pinning_the_web.js

```diff
@@ -63,7 +63,8 @@ PinningTheWeb.prototype = {
   openAndPinSiteFromBrowser: function openAndPinSite(url) {
     this._openUrl(url);
     this._clickPinContextMenu();
-    this.pinSiteButton.tap();
+    // Tap the top left of the pin button.
+    this.pinSiteButton.tap(0, 0);
```

Comment:
I don't understand this change, but I assume there's a good reason for it? Might be worth adding the reason in the comment.